variables.tf

```
@@ -15,6 +15,12 @@ variable "authentication" {
   description = "Whether to protect the cloudfront distribution behind an Okta application"
 }
 
+variable "bucket_force_destroy" {
```

```suggestion
variable "force_destroy" {
```

This make the flag just a little more generic...